dll/win32/shell32/shellrecyclebin/recyclebin_v5.cpp

```diff
@@ -242,26 +246,26 @@ RecycleBin5_RecycleBin5_DeleteFile(
     }
 
     /* Increase INFO2 file size */
-    CloseHandle(s->hInfoMapped);
-    SetFilePointer(s->hInfo, sizeof(DELETED_FILE_RECORD), NULL, FILE_END);
-    SetEndOfFile(s->hInfo);
-    s->hInfoMapped = CreateFileMappingW(s->hInfo, NULL, PAGE_READWRITE | SEC_COMMIT, 0, 0, NULL);
-    if (!s->hInfoMapped)
+    CloseHandle(m_hInfoMapped);
+    SetFilePointer(m_hInfo, sizeof(DELETED_FILE_RECORD), NULL, FILE_END);
+    SetEndOfFile(m_hInfo);
+    m_hInfoMapped = CreateFileMappingW(m_hInfo, NULL, PAGE_READWRITE | SEC_COMMIT, 0, 0, NULL);
+    if (!m_hInfoMapped)
     {
         hr = HRESULT_FROM_WIN32(GetLastError());
         goto cleanup;
     }
 
     /* Open INFO2 file */
-    pHeader = MapViewOfFile(s->hInfoMapped, FILE_MAP_WRITE, 0, 0, 0);
+    pHeader = (PINFO2_HEADER)MapViewOfFile(m_hInfoMapped, FILE_MAP_WRITE, 0, 0, 0);
     if (!pHeader)
     {
         hr = HRESULT_FROM_WIN32(GetLastError());
         goto cleanup;
     }
 
     /* Get number of entries */
-    FileSize.u.LowPart = GetFileSize(s->hInfo, &FileSize.u.HighPart);
+    FileSize.u.LowPart = GetFileSize(m_hInfo, &FileSize.u.HighPart);
```

Comment:
Note: Below (line 278+) there's that GetFileSizeEx call similar to the one I mentioned in your other PR.